chainercv/visualizations/vis_instance_segmentation.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import division
+
+import numpy as np
+
+from chainercv.visualizations.vis_image import vis_image
+from chainercv.visualizations.vis_semantic_segmentation import _default_cmap
+
+
+def vis_instance_segmentation(
+        img, bbox, mask, label=None, score=None, label_names=None,
+        alpha=1, ax=None):
+    """Visualize instance segmentation.
+
+    Example:
+
+        >>> from chainercv.datasets import SBDInstanceSegmentationDataset
+        >>> from chainercv.datasets \
+        ...     import sbd_instance_segmentation_label_names
+        >>> from chainercv.visualizations import vis_image
+        >>> from chainercv.visualizations import vis_instance_segmentation
+        >>> import matplotlib.pyplot as plot
+        >>> dataset = SBDSegmentationDataset()
+        >>> img, bbox, mask, label = dataset[0]
+        >>> vis_instance_segmentation(
+        ...     img, bbox, mask, label,
+        ...     label_names=sbd_instance_segmentation_label_names,
+        ...     alpha=0.7)
+        >>> plot.show()
+
+    Args:
+        img (~numpy.ndarray): An array of shape :math:`(3, H, W)`.
+            This is in RGB format and the range of its value is
+            :math:`[0, 255]`.
+        bbox (~numpy.ndarray): A float array of shape :math:`(R, 4)`.
+            R is the number of objects in the image, and each vector
+            represents a bounding box of an object.
+            The bounding box is :math:`(y_min, x_min, y_max, x_max)`.
+        mask (~numpy.ndarray): A bool array of shape
+            :math`(R, H, W)`.
+            If there is an object, the value of the pixel is :obj:`True`,
+            and otherwise, it is :obj:`False`.
+        label (~numpy.ndarray): An integer array of shape :math:`(R, )`.
+            The values correspond to id for label names stored in
+            :obj:`label_names`.
+        label_names (iterable of strings): Name of labels ordered according
+            to label ids.
+        alpha (float): The value which determines transparency of the figure.
+            The range of this value is :math:`[0, 1]`. If this
+            value is :obj:`0`, the figure will be completely transparent.
+            The default value is :obj:`1`. This option is useful for
+            overlaying the label on the source image.
+        ax (matplotlib.axes.Axis): The visualization is displayed on this
+            axis. If this is :obj:`None` (default), a new axis is created.
+
+    Returns:
+        matploblib.axes.Axes: Returns :obj:`ax`.
+        :obj:`ax` is an :class:`matploblib.axes.Axes` with the plot.
+
+    """
+    if label is not None and not len(bbox) == len(label):
+        raise ValueError('The length of label must be same as that of bbox')
+    if score is not None and not len(bbox) == len(score):
+        raise ValueError('The length of score must be same as that of bbox')
+
+    n_ins_class = len(bbox)
+    colors = [_default_cmap(l) for l in range(n_ins_class + 1)]
```

Comment:
How about `range(1, n_inst)`?
By doing so, you can remove one line.
Also, `np.array` is optional in this case, so we can omit it.